.travis.yml

```diff
@@ -33,18 +35,19 @@ script:
     # So here we are using the approach documented in the issue
     # https://github.com/travis-ci/travis-ci/issues/4190.
     coverage run -m isodatetime.tests &
+    TEST_PID="$!"
     minutes=0
     limit=40
-    while kill -0 $! >/dev/null 2>&1; do
+    while kill -0 "${TEST_PID}" >'/dev/null' 2>&1; do
```

Comment:
More natural to use `ps ...`?

Reply:
+1. And if we are in no hurry, I can give it a try tomorrow, so that I practice some Shell, and then maybe you could review it, please? :grimacing:

Reply:
Oh, this one we already tackled too. So marking as resolved to reduce the number of open conversations in the PR